src/main/java/org/geppetto/core/simulation/ISimulatorCallbackListener.java

```diff
@@ -55,4 +55,5 @@
 	 */
 	void endOfSteps(IAspectConfiguration aspectConfiguration, Map<File, ResultsFormat> results) throws GeppettoExecutionException;
 
+	void externalProcessFailed(String message, Exception e);
```

Comment:
@jrmartin Can you add the comment here which justifies the necessity of this listener interface?